src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Employee/Blocks/employee_options.html.twig

```diff
@@ -0,0 +1,79 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% block employee_options_form %}
+  {{ form_start(employeeOptionsForm, {'action': path('admin_employees_save_options') }) }}
+  <div class="card">
+    <h3 class="card-header">
+      <i class="material-icons">settings</i>
+      {{ 'Employee options'|trans({}, 'Admin.Advparameters.Feature') }}
+    </h3>
+    <div class="card-block row">
+      <div class="card-text">
+        <div class="form-group row">
+          {{ ps.label_with_help(('Password regeneration'|trans({}, 'Admin.Advparameters.Feature')), 'Security: Minimum time to wait between two password changes.'|trans({}, 'Admin.Advparameters.Feature')) }}
+          <div class="col-sm">
+            {{ form_errors(employeeOptionsForm.options.password_change_time) }}
+            {{ form_widget(employeeOptionsForm.options.password_change_time) }}
+
+            {% if not isAllShopContext %}
+            <div class="alert alert-warning mt-2 mb-0" role="alert">
+              <p class="alert-text">
+                {{ 'You can\'t change the value of this configuration field in the context of this shop.'|trans({}, 'Admin.Notifications.Warning') }}
```

Comment:
@LouiseBonnard what would be the correct domain for this key? currently its in `messages`, but i've used `Admin.Notifications.Warning`, is it ok or needs to be changed?

Reply:
Perfect, thanks @sarjon !